packages/flutter_tools/lib/src/base/time.dart

```diff
@@ -32,3 +32,18 @@ class _FixedTimeClock extends SystemClock {
   @override
   DateTime now() => _fixedTime;
 }
+
+/// Format time as 'yyyy-MM-dd HH:mm:ss Z' where Z is the difference between the
+/// timezone of t and UTC formatted according to RFC 822.
+String formatDateTime(DateTime t) {
```

Comment:
I would have assumed that some combination or toUtc() and toIso8601String() would have handled this.

Reply:
Unfortunately, the `DateTime.to*String()` methods only indicate whether the time is UTC or local, but not which timezone or the timezone offset if local.